File: uri 1164 - Perfect Number.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int N,A,S,I;
    scanf("%d",&N);
    while(N>0){
        N--;
        S=0;
        scanf("%d",&A);
        for(I=1;I<A;I++){
            if(A%I ==0)
                S+=I;
        }
        if(S==A)
            printf("%d eh perfeito\n",A);
        else
            printf("%d nao eh perfeito\n",A);
    }
    return 0;
}

File: uri 1164 - Perfect Number.py
n = int(input())
for i in range(n):
    x = int(input())
    total = 0
    for j in range(1, x):
        if x % j == 0:
            total += j
    if total == x:
        print(f"{x} eh perfeito")
    else:
        print(f"{x} nao eh perfeito")
